README.md

```diff
@@ -160,10 +80,90 @@
+- [Frontend Happy Hour Picks](https://github.com/FrontEndHappyHour/front-end-happy-hour-picks)
+- [Programming *expletive*](http://programming-motherfucker.com/)
+- [Wireframes](http://wireframes.tumblr.com/)
+
+# Performance
+- ### Tools
+  - [Mobile-friendly Test from Google](https://search.google.com/test/mobile-friendly?
+  - [PageSpeed Insights from Google](https://developers.google.com/speed/pagespeed/insights/)
+  utm_source=mft&utm_medium=redirect&utm_campaign=mft-redirect)
+  
+# Stay Up to Date
+- [Front End Masters Handbook](https://frontendmasters.com/books/front-end-handbook/2018/)
+- [Frontend Focus](https://frontendfoc.us/)
+- [Hacker News](https://news.ycombinator.com/)
+- [Javascript Weekly](https://javascriptweekly.com/)
+- [Rand Fishkin at Moz](https://moz.com/community/users/63)
+- [Smashing Magazine](https://www.smashingmagazine.com)
+
+# Study Materials
+  - [What is Code](https://www.bloomberg.com/graphics/2015-paul-ford-what-is-code/)
+
+  - ## Javascript
+    - [JavaScript for Cats](https://noblemule.gitbooks.io/javascript-for-cats/content/)
+    - [JavaScript Koans](https://github.com/mrdavidlaing/javascript-koans)
+    - [Learn Javascript](https://gitbookio.gitbooks.io/javascript/)
+    - [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS)
+  - ## Computer Science
+    - [Algorithms](http://ressources.unisciel.fr/algoprog/s00aaroot/aa00module1/res/%5BCormen-AL2011%5DIntroduction_To_Algorithms-A3.pdf)
+    - [Base CS](https://medium.com/basecs)
+    - [Computer Science with Ruby](https://computer-science-one.gitbooks.io/cs101-book/content/)
+
+# Tutorials
+- [Colt Steele's Udemy Web Dev Bootcamp](https://www.udemy.com/the-web-developer-bootcamp/)
+- [Google SEO](https://static.googleusercontent.com/media/www.google.com/en//webmasters/docs/search-engine-optimization-starter-guide.pdf)
+- [Intro to React](https://reactjs.org/tutorial/tutorial.html)
+- [Javascript30](https://javascript30.com/)
+- [Mozilla's Web Dev Tutorials](https://developer.mozilla.org/en-US/docs/Learn)
+- [Project Based Tutorials](https://github.com/tuvtran/project-based-learning)
+- [Salesforce Trailhead](https://trailhead.salesforce.com/)
+- [WebGL](https://webglfundamentals.org/)
+- [Workshopper](https://github.com/workshopper)
+
+# Veteran Tech Benefits
+- [AWS Educate Now Available to U.S. Veterans | AWS Government, Education, & Nonprofits Blo](https://aws.amazon.com/blogs/publicsector/aws-educate-now-available-to-u-s-veterans/)
+- [Free Sales force training for Vets](https://veterans.force.com/NewVetForceHome)
+- [Linkedin for Vets](https://socialimpact.linkedin.com/programs/veterans/premiumform)
+
+# Web Design Inspiration
+- [Brutalist Design](http://brutalistwebsites.com/)
+- [Collect UI](https://medium.com/@CollectUI)
+  
+# Web Security
+- ### Guides
+  - [Awesome Reference Guide from Mozzilla's Infosec Team](https://infosec.mozilla.org/guidelines/web_security.html)
```

Comment:
Also blank space here and the following lines

Reply:
I'm sorry I don't quite get what you mean about this one, where I need to add blank space and the following lines ?